lambda_data_logger/main.py

```diff
@@ -117,6 +119,45 @@ def handler(event, context):
 
     logger.info(f"Uploaded updated {object_name} to S3")
 
+    # GitHub Teams with CoPilot Data
+
+    logger.info("Getting GitHub Teams with CoPilot Data")
+
+    copilot_teams = []
+
+    response = gh.get(f"/orgs/{org}/teams", params={"per_page": 100})
+
+    # Get the last page of teams
+    try:
+        last_page = int(response.links["last"]["url"].split("=")[-1])
+    except KeyError:
+        last_page = 1
+
+    for page in range(1, last_page + 1):
+        response = gh.get(f"/orgs/{org}/teams", params={"per_page": 100, "page": page})
+        teams = response.json()
+        for team in teams:
+            usage_data = gh.get(f"/orgs/{org}/team/{team['name']}/copilot/usage")
+            try:
+                if usage_data.json() != []:
```

Comment:
Small nitpick - Instead of comparing to an empty list you can just do `if usage_data.json()` and will carry on the if block if that array is populated.

Reply:
Done